Graphics/GraphicsEngineOpenGL/src/TextureBaseGL.cpp

```diff
@@ -490,9 +490,23 @@ void TextureBaseGL::CreateViewInternal(const TextureViewDesc& OrigViewDesc, ITex
                     LOG_ERROR_AND_THROW("glTextureView is not supported");
 
                 glTextureView(pViewOGL->GetHandle(), GLViewTarget, m_GlTexture, GLViewFormat, ViewDesc.MostDetailedMip, ViewDesc.NumMipLevels, ViewDesc.FirstArraySlice, NumLayers);
+                
                 DEV_CHECK_GL_ERROR_AND_THROW("Failed to create texture view");
                 pViewOGL->SetBindTarget(GLViewTarget);
 
+                if(ViewDesc.Format == TEX_FORMAT_X24_TYPELESS_G8_UINT || ViewDesc.Format == TEX_FORMAT_X32_TYPELESS_G8X24_UINT)
```

Comment:
It is necessary to check the format support here using `pDeviceGLImpl->GetTextureFormatInfo(ViewDesc.Format)`